tests/testthat/test-object-package.R

```diff
@@ -17,3 +17,94 @@ test_that("person turned into meaningful text", {
     person_desc(comment = c("ORCID" = "1234", "extra"))
   })
 })
+
+test_that("package description not affected if no links", {
+  text <- "A simple description with no links."
+
+  parsed <- package_description_urls(text)
+
+  expect_equal(
+    parsed,
+    text
+  )
+  expect_snapshot(parsed)
+})
+
+test_that("can autolink urls on package Description", {
+  urls <- paste(
+    "<https://github.com/>",
+    "Secured <https://github.com/>.",
+    "No link <www.github.com/>.",
+    "url masked with spaces",
+    "<https://database.ich.org/sites/default/files/Q1E%20Guideline.pdf>",
+    "url fully masked",
+    "<https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=OJ%3AL%3A2015%3A012%3ATOC>"
+  )
+
+  parsed <- package_description_urls(urls)
+
+  expect_equal(
+    parsed,
+    paste(
+      "\\url{https://github.com/}",
+      "Secured \\url{https://github.com/}.",
+      "No link <www.github.com/>.",
+      "url masked with spaces",
+      "\\url{https://database.ich.org/sites/default/files/Q1E\\%20Guideline.pdf}",
+      "url fully masked",
+      "\\url{https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=OJ:L:2015:012:TOC}"
+    )
+  )
+  expect_snapshot(parsed)
```

Comment:
There's no need to include a snapshot when you've already test the correct results about